src/components/ListItem/ListItem.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { cVar, sizes } from '@/styles'
+
+import { ListItemSizes } from '.'
+import { SvgActionCheck } from '../_icons'
+
+const getContainerPadding = (size: ListItemSizes) => {
+  switch (size) {
+    case 'large':
+      return css`
+        padding: ${sizes(3)} ${sizes(4)};
+      `
+    case 'medium':
+      return css`
+        padding: ${sizes(2)} ${sizes(4)};
+      `
+    case 'small':
+      return css`
+        padding: ${sizes(1)} ${sizes(4)};
+      `
+  }
+}
+type ContainerProps = { size: ListItemSizes; hasNodeStart: boolean; disabled?: boolean }
+const disabledStyles = css`
+  opacity: 0.25;
+  cursor: not-allowed;
+`
```

Comment:
hmm I think disabled variants looks different in the designs: 
![image](https://user-images.githubusercontent.com/51168865/144421047-3b074dc4-65d8-4d59-ac95-12d06e6b4946.png)

Reply:
oh, wait, my mistake. I didn't understand the design. It's transparent by default, right?

Reply:
yep